.env

```diff
@@ -0,0 +1,2 @@
+GUNICORN_RELOAD=true
```

Comment:
There's definitely a debate, but it seems a `.env` file should not be version-controlled.